x-pack/test_serverless/functional/test_suites/security/mki_only.ts

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Optional: could rename this file to `index.mki_only.ts` for clarity and consistency with how we do it for other scenarios.

Reply:
Thanks for the suggestion, it makes sense to me.